resources/js/components/module/templates/pieChart/PieChartLegend.vue

```diff
@@ -98,6 +98,8 @@ export default {
     }
   },
   mounted () {
+    this.$refs.canvas.setAttribute('role', 'img')
+    this.$refs.canvas.setAttribute('aria-label', 'Pie chart showing ' + this.chartdata.userData.labels[0] + ', and ' + this.chartdata.sourcesData.labels[0])
```

Comment:
Can update the string to `'Pie chart showing. Your Guess: ' + this.chartdata.userData.labels[0] + ', and Actual Percentage:' + this.chartdata.sourcesData.labels[0]`